lms/envs/aws.py

```diff
@@ -1037,4 +1037,5 @@
 EDEOS_API_KEY = AUTH_TOKENS.get('EDEOS_API_KEY', '')
 EDEOS_API_SECRET = AUTH_TOKENS.get('EDEOS_API_SECRET', '')
 EDEOS_API_URL = AUTH_TOKENS.get('EDEOS_API_URL', '')
+EDEOS_API_STATISTICS = AUTH_TOKENS.get('EDEOS_API_STATISTICS', '')
```

Comment:
auth tokens are for secure data - like passwords. We can put urls to env tokens

Reply:
fixed